tests/test_web/test_module.py

```diff
@@ -208,6 +208,57 @@ async def test_imported_components_can_render_children(display: DisplayFixture):
         assert (await child.get_attribute("id")) == f"child-{index + 1}"
 
 
+async def test_keys_properly_propagated(display: DisplayFixture):
```

Comment:
Can you add a docstring to this that references the issue link?

Reply:
Done.